bindata/v3.11.0/openshift-controller-manager/ingress-to-route-controller-rolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
at the current stage this is not required since we already have the cluster role binding. But we probably should use this binding roleRef to a new basic Role with just the leases